### ISSUE TITLE
Is there any benefit to wrapping this in an AsyncWrapper target or a BufferingWrapper target?

### ISSUE THREAD

**Issue by matthewkrieger** [JDetmar/NLog.Extensions.AzureStorage#8]
Any benefit to wrapping your target in an AsyncWrapper https://github.com/nlog/NLog/wiki/AsyncWrapper-target or BufferingWrapper https://github.com/nlog/NLog/wiki/BufferingWrapper-target target?

**Comment by snakefoot** [comment]
Yes it will allow batching, so it will be able to write multiple LogEvents in a single write-request.

**Comment by matthewkrieger** [comment]
Sorry, to clarify, the NLog.Extensions.AzureStorage target will allow batching – so I *don’t* need the BufferingWrapper target around it, or I *should* use the asyncwrapper/buffering wrapper?

Unfortunately I noticed that this target necessarily creates Append blobs and Azure blobtrigger webjobs don’t support them.

Thank you
